# base path for recordings
save_path : '/home/antovision/catkin_ws/src/AntobotSDK/antobot_datamanager/saved_recordings'
zed_name : 'ZED_LEFT'
service_name : 'anto_rec_left'
